paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/procedure/ExpireSnapshotsProcedure.java

```diff
@@ -42,8 +45,9 @@ public class ExpireSnapshotsProcedure extends BaseProcedure {
                 ProcedureParameter.required("table", StringType),
                 ProcedureParameter.optional("retain_max", IntegerType),
                 ProcedureParameter.optional("retain_min", IntegerType),
-                ProcedureParameter.optional("older_than", TimestampType),
-                ProcedureParameter.optional("max_deletes", IntegerType)
+                ProcedureParameter.optional("older_than", StringType),
+                ProcedureParameter.optional("max_deletes", IntegerType),
+                ProcedureParameter.optional("time_retained", StringType)
```

Comment:
older_than and time_retained? Looks not easy to understand, we should just keep one.

Reply:
Because there is currently compatibility issue. I recommend using a script to address the issue of periodic calls when invoking from the outer layer.

Reply:
> older_than and time_retained? Looks not easy to understand, we should just keep one.

These two parameters can not be used together.

Reply:
> Because there is currently compatibility issue. I recommend using a script to address the issue of periodic calls when invoking from the outer layer.

When using a script to calls periodically, we need to adjust older_than parameter each time. I think it is necessary to keep time_retained,  what do you think?

Reply:
Thanks. @JingsongLi Removed time_retained